src/containers/skills/Skills.js

```diff
@@ -2,25 +2,41 @@ import React from "react";
 import "./Skills.css";
 import SoftwareSkill from "../../components/softwareSkills/SoftwareSkill";
 import { skillsSection } from "../../portfolio";
+import {Fade, Flip} from "react-reveal";
 
 export default function Skills() {
   return (
-    <div className="main" id="skills">
-      <div className="skills-main-div">
-        <div className="skills-image-div">
-          <img alt="Saad Working" src={require("../../assests/images/developerActivity.svg")}></img>
-        </div>
-        <div className="skills-text-div">
-          <h1 className="skills-heading">{skillsSection.title} </h1>
-          <p className="subTitle skills-text-subtitle">{skillsSection.subTitle}</p>
-          <SoftwareSkill />
-          <div>
-            {skillsSection.skills.map(skills => {
-              return <p className="subTitle skills-text">{skills}</p>;
-            })}
-          </div>
-        </div>
-      </div>
-    </div>
-  );
+		<div className="main" id="skills">
+			<div className="skills-main-div">
+				<Flip left duration={2000}>
+					<div className="skills-image-div">
+						<img
+							alt="Saad Working"
+							src={require("../../assests/images/developerActivity.svg")}
+						></img>
+					</div>
+				</Flip>
```

Comment:
Could you change this to Fade left? So it remains consistent with all the other transitions?